module/zfs/zfs_vnops.c

```diff
@@ -4006,13 +4007,14 @@ zfs_rename(struct inode *sdip, char *snm, struct inode *tdip, char *tnm,
  * Insert the indicated symbolic reference entry into the directory.
  *
  *	IN:	dip	- Directory to contain new symbolic link.
- *		link	- Name for new symlink entry.
+ *		name	- Name of directory entry in dip.
  *		vap	- Attributes of new entry.
- *		target	- Target path of new symlink.
- *
+ *		link	- Name for new symlink entry.
  *		cr	- credentials of caller.
  *		flags	- case flags
  *
+ *	OUT:	ipp	- Inode for new symbolic link.
```

Comment:
can we add big comment about inode as Linux specific and provide references like (see comment above) about this specific?
ipp - it is inode Linux specific and known only to persons who looked it before.

Reply:
This is something we'll want to work on once the code can be built on multiple platforms.